index.html

```diff
@@ -1,40 +1,70 @@
 <html>
     <head>
         <title>Tipease II</title>
+        
         <link rel="stylesheet" href="./css/index.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+        <link href="https://fonts.googleapis.com/css?family=Raleway|Source+Sans+Pro" rel="stylesheet">
         <script src="./components/dropdown/dropdown.js"></script>
```

Comment:
If you notice any issues while loading, think about ways that this can be modified or moved to resolve them.

Reply:
Not sure if you will though... but you know what to do just in case.

Reply:
Hm. I'm actually not sure @ryanboris! The only thing I can think of is potentially putting these scripts at the bottom.